blueprints/app/files/package.json

```diff
@@ -36,6 +36,7 @@
     "ember-data": "2.1.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.4",
-    "ember-resolver": "^2.0.3"
+    "ember-resolver": "^2.0.3",
+    "jshint": "^2.8.0"
```

Comment:
JSHint dep was moved down into broccoli-jshint so that it can properly use the JSHint API's without worrying about getting an invalid version.  I'd prefer to not add this back at the project level again.